static/app/components/onboardingWizard/taskConfig.tsx

```diff
@@ -142,7 +142,7 @@ export function getOnboardingTasks({
       task: OnboardingTaskKey.FIRST_EVENT,
       title: t('Capture your first error'),
       description: t(
-        "Time to test it out. Now that you've created a project, capture your first error. We've got an example you can fiddle with."
+        "Now that you've created a project, test it by capturing your first error. We've got an example you can fiddle with."
```

Comment:
maybe just "Capture your first error."

Reply:
How about: "Now that you've created a project, capture your first error. We've got...."  (If we just leave it as "Capture your first error." It'll be too repetitive with the title. :)

Reply:
🤦 duh. sounds good. thank you.